elife_bus_sdk/queues/sqs_queue.py

```diff
@@ -0,0 +1,93 @@
+from typing import Dict, Generator, List
+
+import boto3
+
+from elife_bus_sdk.events import Event
+from elife_bus_sdk.queues.message_queue import MessageQueue
+
+
+class SQSMessageQueue(MessageQueue):
+    name = 'sqs'
+
+    def __init__(self, **overrides):
+        self._queue_name = overrides.pop('queue_name', None)
+        self._overrides = overrides
+        self._resource = boto3.resource(self.name, **self._overrides)
+        self._queue = self._resource.get_queue_by_name(QueueName=self._queue_name)
+
+    def dequeue(self) -> List['sqs.Message']:
```

Comment:
Have changed `dequeue` so that it implements returning the `Event` objects, all `poll` now does is call `dequeue` (no additional parsing). This allows the caller to use `dequeue` or `poll` and get consistent return values.